lexicons/app/bsky/system/declRef.json

```diff
@@ -0,0 +1,18 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.system.declRef",
```

Comment:
I'd kinda like for the interface here to more resemble a `strongRef`. references to declaration are most often used in a similar manner to strongRef as a `(did, declarationCid)` tuple. in most cases we don't actually need the `actorType` in that context, just an integrity check that it didn't change